ui: add missing tooltips

vladlos opened the pull request:
created/updated all needed tooltips for Cluster Overview, Metrics and Databases pages listed in #51835 and added corresponding storybooks.

Resolves: #50095, #51835

Release note (ui): none

<img width="1277" alt="Screenshot 2020-07-30 at 18 42 12" src="https://user-images.githubusercontent.com/12850886/88943918-c3c9da80-d294-11ea-8300-6e0bc3c0ee69.png">

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/51921)
<!-- Reviewable:end -->

dhartunian commented on pkg/ui/src/views/cluster/containers/nodeOverview/tooltips.tsx:
```diff
@@ -0,0 +1,164 @@
+                </p>
+                <p>
+                    <Anchor href={capacityMetrics} target="_blank">
+                        How is this metrics calculated?
```
Minor nit, change to "metric"

dhartunian reviewed (approved):
LGTM. One small spelling nit, otherwise good to go

dhartunian reviewed (approved):
Excellent! Thanks for wrapping up all the changes in here! <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/51921#-:-MDoapcQ9lCym1HNr-Vc:bamsoyc)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @elkmaster and @koorosh)



<!-- Sent from Reviewable.io -->

vladlos committed:
ui: add missing tooltips

created/updated all needed tooltips and titles for Cluster Overview, Metrics and Databases pages listed in #51835 and added corresponding storybooks.

Resolves: #50095, #51835

Release note (ui): none

elkmaster commented:
bors r+
